eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -49,7 +49,11 @@
 
 /**
  * Test case for {@link OptimizeMojo}.
- *
+ * @todo #2638:15min The test {@link #doesNotCrashesOnError(Path path)} has the mistake:
+ *  Here "Matchers.hasKey(String.format("target/%s/foo/x/main.%s",
+ *  ParseMojo.DIR, TranspileMojo.EXT)".
+ *  But it should be "Matchers.hasKey(String.format("target/%s/foo/x/main.%s",
+ *  OptimizeMojo.DIR, TranspileMojo.EXT)". Or is this not a mistake?
```

Comment:
@Yanich96 seems you're right about the mistake, but I believe it's not big enough to create a puzzle for it. Could you please fix it?

Reply:
@maxonfjvipon ok